mautic/mautic
# [UX] Specify what is the Default order when creating a webhook

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
The purpose of this PR is to enhance the webhook creation process by providing additional context for the ‘Default’ order option. A tooltip has been implemented to inform users that the ‘Default’ setting is determined by the system settings. This small yet significant change ensures users are fully aware of how the default order operates, fostering a more informed and efficient user experience. This update is particularly advantageous for new users or those configuring webhooks for the first time, as it demystifies the default behavior and integrates seamlessly with the existing system configuration.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Webhooks
3. Create new
4. Hover over Order
5. Look at the tooltip

![image](https://github.com/mautic/mautic/assets/116097999/a69845be-5bca-400f-a891-5e2f8597b086)
